Package.swift

```diff
@@ -743,3 +749,11 @@ let package = Package(
         )
     ]
 )
+
+if ProcessInfo.processInfo.environment["STRICT_CONCURRENCY"] == "true" {
+    for target in package.targets {
+        if !target.isTest {
+            target.swiftSettings = [.unsafeFlags(["-Xfrontend", "-warn-concurrency"])]
```

Comment:
Shouldn't this be strict concurrency rather than warning
```suggestion
            target.swiftSettings = [.unsafeFlags(["-Xfrontend", "-strict-concurrency=complete"])]
```

Reply:
I didn't know about that one. Finding these flags is so bloody hard work, having them as errors makes life a lot easier